common/customizable_localization/00_divinity_custom_loc.txt

```diff
@@ -567,56 +567,548 @@ GetActualBishopTitle = {
+				religion = religion:secular_group
+			}
+		}
+		localization_key = councillor_court_chaplain_secular_religion_county
+	}
+	# Arcane
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_kingdom
+			religion = religion:arcane_group
 			is_female = no
 		}
-		localization_key = councillor_court_chaplain_generic_kingdom
+		localization_key = councillor_court_chaplain_arcane_religion_kingdom
 	}
 	text = {
 		trigger = {
 			liege.highest_held_title_tier >= tier_kingdom
+			religion = religion:arcane_group
 			is_female = yes
 		}
-		localization_key = councillor_court_chaplain_generic_kingdom_female
+		localization_key = councillor_court_chaplain_arcane_religion_kingdom_female
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_duchy
+			religion = religion:arcane_group
+			is_female = no
+		}
+		localization_key = councillor_court_chaplain_arcane_religion_duchy
 	}
 	text = {
 		trigger = {
 			liege.highest_held_title_tier >= tier_duchy
+			religion = religion:arcane_group
+			is_female = yes
 		}
-		localization_key = councillor_court_chaplain_generic_duchy
+		localization_key = councillor_court_chaplain_arcane_religion_duchy_female
 	}
 	text = {
 		trigger = {
 			liege.highest_held_title_tier < tier_duchy
+			religion = religion:arcane_group
+			is_female = no
 		}
-		localization_key = councillor_court_chaplain_generic_county
+		localization_key = councillor_court_chaplain_arcane_religion_county
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier < tier_duchy
+			religion = religion:arcane_group
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_arcane_religion_county_female
+	}
+	# Dragon
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_kingdom
+			faith = faith:sect_of_the_dragons
+			is_female = no
+		}
+		localization_key = councillor_court_chaplain_dragon_religion_kingdom
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_kingdom
+			faith = faith:sect_of_the_dragons
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_dragon_religion_kingdom_female
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_duchy
+			faith = faith:sect_of_the_dragons
+			is_female = no
+		}
+		localization_key = councillor_court_chaplain_dragon_religion_duchy
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_duchy
+			faith = faith:sect_of_the_dragons
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_dragon_religion_duchy_female
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier < tier_duchy
+			faith = faith:sect_of_the_dragons
+			is_female = no
+		}
+		localization_key = councillor_court_chaplain_dragon_religion_county
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier < tier_duchy
+			faith = faith:sect_of_the_dragons
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_dragon_religion_county_female
+	}
```

Comment:
Faith-specific names should be above religion-specific.
Draconic chaplains are called scholars. And since Titanic names are above Draconic, the game use Titanic ones.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/100632100-dc7ef580-3345-11eb-91bc-d1c6524dddf8.png)

</details>